sdk/src/Core/Amazon.Util/EC2InstanceMetadata.cs

```diff
@@ -26,6 +26,8 @@
 using Amazon.Runtime.Internal.Util;
 using AWSSDK.Runtime.Internal.Util;
 using Amazon.Runtime.Internal;
+using Amazon.Util.Internal;
+using System.Text.Json.Serialization.Metadata;
```

Comment:
Is `NET6_0_OR_GREATER` not required here? You added it in the new class (https://github.com/aws/aws-sdk-net/blob/dd0fc7a34fd787d804b6403db45a651dc028e44a/sdk/src/Core/Amazon.Util/Internal/JsonSerializerContexts.cs#L5-L9)

Reply:
I forgot to recompile framework which was failing because of the `using System.Text.Json.Serialization.Metadata;`. I have now removed the unnecessary `using`.